### ISSUE TITLE
Bug.. cann't upload file via upload button to subfolder. Picture still ends in root folder.

### ISSUE THREAD

**Issue by nickn17** [DevCrh/kcfinder-Resurrected#1]
This fixed issue 
js/060.toolbar.js

_.uploadFile = function (form) {
    if (!_.dirWritable) {
        _.alert(_.label("Cannot write to upload folder."));
        $('#upload').detach();
        _.initUploadButton();
        return;
    }

// add this line here
$('form input[name=dir]').val(_.dir);
....

**Comment by DevCrh** [comment]
Thank you the bug has already been fixed, I appreciate very much the cooperation and interest in the project
